site/src/content/docs/components/breadcrumb.mdx

```diff
@@ -83,3 +85,17 @@ Breadcrumb will never wrap, the number of displayed items depends on the availab
       <li class="breadcrumb-item active" aria-current="page"><span title="Very long page to show off responsiveness behavior">Very long page to show off responsiveness behavior</span></li>
     </ol>
   </nav>`} />
+
+### Avoiding truncation
+
+To prevent a specific item from being truncated, add `.flex-shrink-0` on the `<li>`. The penultimate item has this option set by default so it's not useful to add it. Be aware that adding this utility to all items would result in an overflow. The items will still be hidden on smaller screens.
```

Comment:
Maybe precise a use case or warn a bit more about the misuses of this. Maybe add that brands need to not be cut or something like this.